CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ add_subdirectory(util)
 zephyr_library_link_libraries(MCUMGR)
 
 target_link_libraries(MCUMGR INTERFACE
-  zephyr_interface
+  ${IMAGE}zephyr_interface
```

Comment:
We haven't needed the IMAGE prefix for a while now, since @tejlmand rewrote the multi-image patch set. Should be safe to delete.

Reply:
wonder for how long time those will pop up.
I caught one like that last week in an very old PR that had been rebased.
This is safe to remove.

Reply:
prefix removed